templates/donation.html

```diff
@@ -1,6 +1,20 @@
 {% extends "template.html" %}
 
 {% block content %}
-    <p>In the development of the site, you can scan the QR code to support us.</p>
-        <img width="400px" height="400px" src="/static/img/mm_facetoface_collect_qrcode_1546415955505.png">
-{% endblock %}
```

Comment:
Do you think excluding the original developers from their work is a responsible thing ?

Reply:
i don't think that, but i thought teacher want us to make the work be ours, i'll fix it.

Reply:
@ileaderx @hema-001 

Hi guys

Thanks for your work.

We should include everyone (as far as we know) who has contributed to this work, perhaps in reverse chronological order.

**Hui**